### PR TITLE
Add moment banner variants

### PR DESCRIPTION
**Pull request by johnduffell**
## What does this change?
Add all the different variants of the banner

This PR adds different variants for:
UK potential
Elsewhere potential (2 variants)
Everywhere existing

This a WIP and will be refined... @jranks123 is working on the styling and @tomrf1 is looking at the logic

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        {
+            id: 'control',
+            test: (): void => {}, // banner tests look at the bucket and vary the copy themselves
+            engagementBannerParams: {
+                messageText: thankYouCopy,
+                // buttonCaption?: string, TO be decided with non engineers
+                template: acquisitionsBannerFivTemplate,
+                bannerModifierClass: 'fiv-banner',
+                minArticlesBeforeShowingBanner: 0,
+                userCohort: 'OnlyExistingSupporters',
```
Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner.js]: at the moment it doesn't put people into this test, because the users get into the earlier tests based on being either in or not in GB and this one isn't looked at.  This will be in another PR

File: static/src/javascripts/__flow__/types/acquisitions.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ declare type AcquisitionsEpicTemplateCopy = {
 };
 
 declare type EngagementBannerTemplateParams = {
+    leadSentence: string,
```
Comment (johnduffell) [inline static/src/javascripts/__flow__/types/acquisitions.js]: image

Comment (johnduffell) [reply static/src/javascripts/__flow__/types/acquisitions.js]: we haven't yet added in the main text above, as we are not sure if it will be an SVG list or a string (re @paperboyo @GHaberis @ionamckendrick  discussion about font download times )

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ export const acquisitionsBannerFivTemplate = (
         </div>
         <div class="fiv-banner__container">
             <div class="fiv-banner__text">
-                HIHIHI ${params.messageText}${params.ctaText}
+                HIHIHI ${params.messageText}${params.leadSentence}
```
Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js]: @jranks123 is already editing this file so I haven't changed any of the placeholder stuff

File: static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner.js [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    author: 'John Duffell',
+    description: 'test copy on engagement banner outside UK for the feb moment',
+    audience: 1,
+    audienceOffset: 0,
+    successMeasure: 'AV per impression',
+    audienceCriteria: 'All',
+    idealOutcome: 'best AV possible',
+    canRun: () => getSync() !== 'GB',
+    showForSensitive: true,
+    campaignId: 'february_moment', // TODO check this matches the one used elsewhere
```
Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner.js]: need to change to empower_campaign or any other suggesiton that comes through

File: static/src/javascripts/__flow__/types/acquisitions.js (hunk 2 above)
Comment (tomrf1) [inline static/src/javascripts/__flow__/types/acquisitions.js]: should this field be optional? Not all banner templates will use it

Comment (johnduffell) [reply static/src/javascripts/__flow__/types/acquisitions.js]: it will just get empty string, but maybe empty is better

Comment (johnduffell) [reply static/src/javascripts/__flow__/types/acquisitions.js]: done that! thanks

**Review by tomrf1 — APPROVED** [review]
